recipes/use_cases/end2end-recipes/raft/README.md

```diff
@@ -0,0 +1,241 @@
+
+## Introduction:
```

Comment:
```suggestion
## Chatbot Recipe:

With the release of the Llama family of models, we've seen more businesses wanting to adapt them for specific uses to better serve their customers and cut operation costs. A popular use case is developing a customer service chatbot for companies with extensive domain-specific data.
In this tutorial, we'll provide an end-to-end guide for building a domain-specific chatbot from scratch. We'll use a Llama chatbot as a case study, focusing on a chatbot that answers questions about Llama models. While the Meta Llama 3 70B Instruct model offers strong reasoning and knowledge, it can be expensive to use. To save costs, we'll focus on the Meta Llama 8B Instruct model, aiming to achieve similar accuracy at lower costs.
Our main focus will be on techniques, tools, and pipelines for data generation, evaluation, and reducing hallucination using [RAFT](https://arxiv.org/abs/2403.10131) by Tianjun Zhang et al from UC Berkeley.

```

Reply:
changed!